lambdas/es/indexer/index.py

```diff
@@ -99,6 +99,7 @@ def append(
             version_id
     ):
         """format event as a document and then queue the document"""
+        more_meta = transform_meta(meta or {})
```

Comment:
can you rename this to something clearer than `more_meta`? `processed_meta` or something like that? even `obj_meta` makes a little more sense.